docs/src/content/docs/de/404.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Nicht gefunden
+template: splash
+editUrl: false
+hero:
+  title: '404'
+  tagline: <strong>Houston, wir haben ein Problem.</strong>Wir konnten diese Seite nicht finden.<br>Prüfe die URL oder verwende die Suchleiste.
```

Comment:
Missing a space here after the closing `</strong>` tag:
<img width="134" alt="image" src="https://github.com/withastro/starlight/assets/357379/a7d8e3b6-344a-4c9c-bd90-75adee4e5deb">


```suggestion
  tagline: <strong>Houston, wir haben ein Problem.</strong> Wir konnten diese Seite nicht finden.<br>Prüfe die URL oder verwende die Suchleiste.
```